integration/sawtooth_integration/tests/test_intkey_cli.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+import unittest
+import time
+import logging
+import subprocess
+import shlex
+
+from sawtooth_integration.tests.integration_tools \
+    import wait_for_rest_apis, RestClient
+
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.INFO)
+
+
+# the appropriateness of these parameters depends on
+# the rate at which blocks are published
```

Comment:
Very vague comment - could you be more specific? If not, at least a nod to how to determine the numbers you want. Not going to block on this though.